AutomationHelpers/src/UserCodeCollections/Utils.cs

```diff
@@ -26,5 +26,14 @@ public static void ReportException(Exception exception, string source)
                 exception.GetFullMessage(),
                 new SimpleReportMetadata("stacktrace", exception.StackTrace));
         }
+
+        public static string CreateRelativePath(string source, string target)
```

Comment:
Missing CheckArgumentNotNullOrEmtpy checks

Reply:
Who are you again? 🤔 
😄 Great point! Will do!